src/charm.py

```diff
@@ -12,7 +12,7 @@
 import logging
 from typing import Dict, List
 
-from charms.tls_certificates_interface.v2.tls_certificates import (  # type: ignore[import]
+from charms.tls_certificates_interface.v2.tls_certificates import (  # type: ignore
```

Comment:
```suggestion
from charms.tls_certificates_interface.v2.tls_certificates import (  # type: ignore[import-not-found]  # noqa: E501
```

Avoid generic "ignore" statements.